examples/meta/src/multiclass/multiclass_machine.sg.in

```diff
@@ -0,0 +1,36 @@
+CSVFile f_feats_train("@SHOGUN_DATA@/classifier_4class_2d_linear_features_train.dat")
```

Comment:
can I suggest we name this "one_versus_rest" actually?

Reply:
this is not addressed. I meant the filename of the example. The current filename is not really good as it contains the folder name and it doesnt actually say what we do in here (one_versus_rest)